src/Controller/UI/ReportDatasetSubmission.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Controller\UI;
+
+use App\Entity\Dataset;
+use App\Entity\DatasetSubmission;
+use App\Entity\PersonDatasetSubmission;
+use App\Form\DatasetSubmissionType;
+use App\Form\PersonDatasetSubmissionType;
+use App\Repository\DatasetRepository;
+use Doctrine\Common\Collections\Collection;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * A controller for a Report of Research Groups and related Datasets.
+ *
+ * @return Response A Symfony Response instance.
+ */
+class ReportDatasetSubmission extends ReportController
+{
+    /**
+     * The default action for Dataset Review.
+     *
+     * @param string            $udi               A UDI.
+     * @param DatasetRepository $datasetRepository The dataset repository.
+     *
+     * @throws NotFoundHttpException When the dataset does not exist.
+     *
+     *
+     * @Route("/overview/{udi}")
```

Comment:
No route name

Reply:
Route name is optional, it makes a "magic" route from the class, controller, action.